win32ss/user/ntuser/desktop.c

```diff
@@ -1516,7 +1496,7 @@ DesktopWindowProc(PWND Wnd, UINT Msg, WPARAM wParam, LPARAM lParam, LRESULT *lRe
         }
         default:
             TRACE("DWP calling IDWP Msg %d\n",Msg);
-            //*lResult = IntDefWindowProc(Wnd, Msg, wParam, lParam, FALSE);
+            *lResult = IntDefWindowProc(Wnd, Msg, wParam, lParam, FALSE);
```

Comment:
I am not sure if it is appropriate to enable this. This window proc runs in the desktop thread which is a weird environment in general.